src/compass/utils/h5_helpers.py

```diff
@@ -80,11 +80,13 @@ def add_dataset_and_attrs(group, meta_item):
 
 
 def init_geocoded_dataset(grid_group, dataset_name, geo_grid, dtype,
-                          description, data=None):
+                          description, data=None, output_cfg=None):
     '''
     Create and allocate dataset for isce.geocode.geocode_slc to write to that
-    is NC compliant
+    is NetCDF compliant
```

Comment:
small nitpick which you could ignore... i think it's CF-compliant... but not actually "NetCDF compliant" becasue netcdf right now doesnt allow complex data? I could be wrong